build/configs/stm32l4r9ai-disco/pm/defconfig

```diff
@@ -538,8 +538,9 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
```

Comment:
Change the title to os/arch/arm/src/common : Change the var name from this to that ti fix the build breaks or something